dali/python/CMakeLists.txt

```diff
@@ -15,15 +15,15 @@
 # Get all the srcs
 collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 collect_sources(DALI_PYTHON_BACKEND_SRCS)
-pybind11_add_module(${dali_python_lib} MODULE ${DALI_PYTHON_BACKEND_SRCS})
+pybind11_add_module(backend_impl MODULE ${DALI_PYTHON_BACKEND_SRCS})
```

Comment:
Why not rename it to something more specific, like... `dali_python`? backend_impl is far too generic - it's confusing even at our own cmake level, but it will be a terrible disfavor to users to have a library with that non-specific name installed by a _very_ specific library.

Reply:
Done